lib/communicationUtilities/src/main/java/org/sagebionetworks/utils/MD5ChecksumHelper.java

```diff
@@ -1,113 +1,14 @@
 package org.sagebionetworks.utils;
 
-import java.io.ByteArrayInputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.UnsupportedEncodingException;
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
+import java.util.regex.Pattern;
 
-/**
- * Code lifted from http://www.rgagnon.com/javadetails/java-0416.html
- * 
- * Found similar code later in org/apache/commons/codec/digest/DigestUtils.java
- * 
- * @author deflaux
- * 
- */
 public class MD5ChecksumHelper {
```

Comment:
This class no longer calculates MD5 checksums. The only remaining method verifies that a string is a valid (not necessarily correct) MD5 checksum

Reply:
Probably Rename to MD5ChecksumUtil